lib/Dashboard/CalendarWidget.php

```diff
@@ -143,31 +154,65 @@ public function load(): void {
 	 * @param int $limit  Max 14 items is the default
 	 */
 	public function getItems(string $userId, ?string $since = null, int $limit = 7): array {
+		// This is hw JS does it:
+		//          const start = dateFactory()
+		//			const end = dateFactory()
+		//			end.setDate(end.getDate() + 14)
+		//          const startOfToday = moment(start).startOf('day').toDate()
+		// get all vevents in this time range
+		// if "show tasks" is enabled, get all todos in the time range
+		// sort events by time
+		// filter events by:
+		// .filter(event => !event.classNames.includes('fc-event-nc-task-completed'))
+		// .filter(event => !event.classNames.includes('fc-event-nc-cancelled'))
```

Comment:
**issue:** This part is no longer taken into account.

Reply:
![yoda-patience](https://user-images.githubusercontent.com/7427347/217779268-783c4aee-1813-468d-b775-9c5ce5564e44.gif)